homeassistant/components/number/__init__.py

```diff
@@ -49,6 +51,18 @@ async def async_setup(hass: HomeAssistantType, config: ConfigType) -> bool:
         "async_set_value",
     )
 
+    component.async_register_entity_service(
```

Comment:
Please open an architecture issue first for discussion and approval.

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
https://github.com/home-assistant/architecture/issues/484